src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -14,10 +14,10 @@
     <sessionState mode="Off" />
   </system.web>
   <appSettings>
-    <!--
-    <add key="Local.Lucene.Directory" value="C:\nuget\SearchData\Lucene-v2v3" />
+
```

Comment:
Commented because it isn't used in deployed env. Uncommented to test locally. Reverted.

---
In reply to: [75556589](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75556589) [](ancestors = 75556589)